src/GridFS/CollectionWrapper.php

```diff
@@ -266,7 +263,7 @@ public function insertFile(array|object $file): void
     /**
      * Updates the filename field in the file document for all the files with a given filename.
      */
-    public function updateFilenameForFilename(string $filename, string $newFilename): ?int
+    public function updateFilenameForFilename(string $filename, string $newFilename): int
```

Comment:
The class is internal, we can change the return type. This is always an `int`, an exception is thrown in case of error.